package.json

```diff
@@ -18,7 +18,8 @@
   "module": "dist-es6/index.js",
   "files": [
     "src",
-    "dist"
+    "dist",
+    "dist-es6"
```

Comment:
To minimize risk of things going wrong, I am not sure that this change should be part of the same PR. This will cause webpack to start using a different dist that doesn't have the transform, increasing the test matrix. 

Note that `dist-es6` is not pure ES6, it is completely transpiled to ES5 **except** import/exports, so it could still be affected by babel issues.

Reply:
Happy to drop it from this PR.
While I am not sure if the `module` field is actually effective, what is the point of building to es6 if the transpiled code is not included in the package?